### PR TITLE
Use haskell ci

### PR DESCRIPTION
**Pull request by chessai**

### CODE REVIEW

**Comment by fosskers** [comment]
Ah you beat me to it.

**Comment by fosskers** [comment]
Do we still want Travis instead of Github CI?

File: streaming-bytestring.cabal [hunk 1]
```diff
@@ -178,6 +178,8 @@ cabal-version:       >=1.10
 stability:           Experimental
 homepage:            https://github.com/haskell-streaming/streaming-bytestring
 bug-reports:         https://github.com/haskell-streaming/streaming-bytestring/issues
+tested-with: GHC==7.8.4, GHC==7.10.3, GHC==8.0.2, GHC==8.2.2, GHC==8.4.4, GHC==8.8.4, GHC==8.10.1
+
```
Comment (vdukhovni) [inline streaming-bytestring.cabal:182]: Just curious whether the omission of `8.6.5` is deliberate?  Perhaps it is not sufficiently different from one or the other of `8.4.4` and `8.8.4` to bother spending extra time on tests?  Or should it be in the test "matrix"?

Comment (fosskers) [reply streaming-bytestring.cabal:182]: Probably a mistake, I'll check them.

Comment (vdukhovni) [reply streaming-bytestring.cabal:182]: Maybe also test 8.10.2 instead of 8.10.1 (assuming available in the CI toolchain).

**Pull request merged**